### ISSUE TITLE
MC 1.18 unrecognized

### ISSUE THREAD

**Issue by SlimeDog** [Photon-GitHub/AACAdditionPro#478]
Spigot 1.18-3350
AACAdditionPro v7.1.3

```
[22:47:15] [Server thread/INFO]: [AACAdditionPro] Enabling AACAdditionPro v7.1.3
[22:47:15] [Server thread/ERROR]: Error occurred while enabling AACAdditionPro v7.1.3 (Is it up to date?)
java.lang.ExceptionInInitializerError: null
        at de.photon.aacadditionpro.AACAdditionPro.onEnable(AACAdditionPro.java:73) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
        at net.lenni0451.spm.utils.PluginUtils.enablePlugin(PluginUtils.java:98) ~[?:?]
        at net.lenni0451.spm.utils.PluginUtils.loadPlugin(PluginUtils.java:220) ~[?:?]
        at net.lenni0451.spm.commands.subs.Load_Sub.execute(Load_Sub.java:54) ~[?:?]
        at net.lenni0451.spm.commands.PluginManager_Command.lambda$onCommand$0(PluginManager_Command.java:94) ~[?:?]
        at net.lenni0451.spm.commands.PluginManager_Command.onCommand(PluginManager_Command.java:110) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.dispatchCommand(CraftServer.java:800) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3350-Spigot-dbf4938-1825cc0]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.dispatchServerCommand(CraftServer.java:785) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3350-Spigot-dbf4938-1825cc0]
        at net.minecraft.server.dedicated.DedicatedServer.bf(DedicatedServer.java:453) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3350-Spigot-dbf4938-1825cc0]
        at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:429) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3350-Spigot-dbf4938-1825cc0]
        at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1206) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3350-Spigot-dbf4938-1825cc0]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1034) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3350-Spigot-dbf4938-1825cc0]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:304) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3350-Spigot-dbf4938-1825cc0]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: de.photon.aacadditionpro.exception.UnknownMinecraftException: Unknown minecraft version.
        at java.util.Optional.orElseThrow(Optional.java:403) ~[?:?]
        at de.photon.aacadditionpro.ServerVersion.<clinit>(ServerVersion.java:47) ~[?:?]
        ... 19 more
```

**Comment by Photon-GitHub** [comment]
Minecraft 1.18 support in 7.1.4

**Comment by SlimeDog** [comment]
Confirmed fixed in AACAdditionPro 7.1.4. Thanks.
